interim-jan-2021/slides-directories.md

```diff
@@ -0,0 +1,30 @@
+# Scenario 3: Directories of Encrypted Resolvers
+
+Three Parties:
+* Publisher: Curates a list of distinct resolvers
+* Client: Fetches the list from a trusted source
+* Resolvers: Identified in the list, ready for access by the client
+------------------------------------------------
+# Example use cases
+
+* A browser vendor wants to provide users with a list of resolvers to consider, curated by a trusted third party
+* An OS vendor wants to keep its list of trusted resolvers current without requiring a software update
+* A user wants to choose a resolver from a list offered by a network operator who they trust
```

Comment:
From @jim-add

```suggestion
* A user wants to choose a resolver from a list offered by a network operator who they trust
* A network operator/manager wants to provide a list of resolvers to local devices/users (think DHCP on steroids but not necessarily using DHCP)
```

Reply:
@jim-add, what's the difference between these two bullets?  They look very similar to me.  What are you supposing the client will do with this list of resolvers?